lib/sax.js

```diff
@@ -415,7 +415,7 @@ function parseElementStartPart(source, start, el, currentNSMap, entityReplacer,
 }
 
 /**
- * @return true if has new namespace define
+ * @returns True if has new namespace define.
```

Comment:
I assume the following makes more sense/is easier to understand?
```suggestion
 * @returns `true` if a new namespace has been defined.
```